tracing-serde/src/lib.rs

```diff
@@ -183,7 +183,9 @@ impl<'a> Serialize for SerializeRecord<'a> {
     }
 }
 
-struct SerdeMapVisitor<S: SerializeMap> {
+/// Implements `tracing_core::field::Visit` for some `serde::ser::SerializeMap`.
+#[derive(Debug)]
+pub struct SerdeMapVisitor<S: SerializeMap> {
```

Comment:
I'm okay with making this public, but if we're going to, we probably ought to make `SerdeStructVisitor` public as well?

Reply:
I can go ahead and do this as well.